modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,216 @@
+[[typography_default_lang]]
+== `+typography_default_lang+`
+
+This option gives the user the ability to pre-set a default typography language within the {productname} editor. By setting up the `typography_default_lang`, when the user opens the {productname} editor for the first time, it will automatically default to the new default language applied. If this default language is subesequently changed by the user, the new default will be saved and persistent between documents.
```

Comment:
```suggestion
This option gives the user the ability to pre-set a default typography language within the {productname} editor. By setting up the `typography_default_lang`, when the user opens the {productname} editor for the first time, it will automatically default to the new default language applied. If this default language is subsequently changed by the user, the new default will be saved and persistent between documents.
```